src/NUnitFramework/nunitlite.tests/nunitlite.tests-2.0.csproj

```diff
@@ -82,6 +82,10 @@
     <Compile Include="TokenizerTests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="NUnit.System.Linq, Version=0.6.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\NUnit.System.Linq.0.6.0\lib\net20\NUnit.System.Linq.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
If we aren't adding the feature to the 2.0 build, then the tests should be excluded as well. In that case, we shouldn't need NUnit.System.Linq, should we?